pkg/manager/manager.go

```diff
@@ -42,6 +42,7 @@ type Manager struct {
 	SouthboundErrorChan     chan events.DeviceResponse
 	Dispatcher              dispatcher.Dispatcher
 	ModelRegistry           map[string]ModelPlugin
+	ModelReadOnlyPaths      map[string][]string
```

Comment:
I'm in doubt this information is in the right place here, is it worth creating a model registry store that encompasses both the elements at line 44 and at line 45 ?

Reply:
Good question: **manager** is the one and only global object - I wasn't inclined to create another - didn't see the point.
It doesn't require a store - it's just a cache.
We could do one object (struct) that encompasses both though if you think that would do.

Reply:
I'm fine leaving as is for now.